geo/src/algorithm/relate/geomgraph/index/mod.rs

```diff
@@ -1,13 +1,20 @@
 mod edge_set_intersector;
-mod prepared_geometry;
 mod rstar_edge_set_intersector;
 mod segment;
 mod segment_intersector;
 mod simple_edge_set_intersector;
-
+#[deprecated(
+    since = "0.31.1",
```

Comment:
We don't yet know whether we'll cut 0.31.1 or 0.32.0 next (probably 0.31.1?), so not sure what the best practice is here.

Reply:
I think what you've done is the most reasonable.

Adding the key word "deprecated" into the release notes might be a helpful hint to look for such a thing, if it in fact becomes a breaking release, so we can update this.